apps/fluent-tester/src/RNTester/TestComponents/Button/ButtonFocusTest.tsx

```diff
@@ -13,7 +14,10 @@ export const ButtonFocusTest: React.FunctionComponent<{}> = () => {
   const onFocus = React.useCallback(() => {
     setState({ focused: !state.focused });
     if (buttonRef.current && !state.focused) {
+      const node = findNodeHandle(buttonRef.current);
```

Comment:
Would be nice if we can make this an actual test w/ Sammy's E2E changes.  I'd remove the console.log but it's nbd

Reply:
Good Idea! I'll create a work item for it for separate check in.